spyder/plugins/switcher/widgets/item.py

```diff
@@ -0,0 +1,399 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Switcher Item Widget."""
+
+# Standard library imports
+import os
+import sys
+
+# Third party imports
+from qtpy.QtCore import (QSize, Qt)
+from qtpy.QtGui import QStandardItem, QTextDocument
+from qtpy.QtWidgets import QApplication
+
+# Local imports
+from spyder.config.base import _
+from spyder.config.manager import CONF
+from spyder.config.utils import is_ubuntu
+from spyder.py3compat import TEXT_TYPES, to_text_string
```

Comment:
Seems like a couple of unused imports could be removed here:
```suggestion
from spyder.config.utils import is_ubuntu
from spyder.py3compat import to_text_string
```